packages/atomic/src/index.html

```diff
@@ -106,6 +107,68 @@
         margin: 10px 0;
       }
 
+      @media only screen and (max-width: 1024px) {
```

Comment:
Could we use some new atomic variables to ensure the breakpoints users would be able to define in their page fit with the one we use inside the components?

https://tailwindcss.com/docs/breakpoints

Reply:
Yes, great idea !

Reply:
Hum apparently you cannot do something like this: 

```
 @media only screen and (max-width: var(--atomic-breakpoint-lg)) {
[...]
}
```

https://stackoverflow.com/a/40723269

I can still define variable though, so that you can modify them for components

Reply:
Well, actually, never mind... this does not work with things like: 

```
 screens: {
        "sm": "var(--atomic-breakpoint-sm)",
```

This also generates an invalid css selector... 🤔  

We'll have to figure this one out later ....